app/assets/v2/js/pages/bounty_details.js

```diff
@@ -337,12 +340,30 @@ window.addEventListener('load', function() {
                     actions.push(entry);
                     if(result['status']=='open' && !isBountyOwner(result) ){
                         var entry = {
-                            href: '/funding/claim?source='+result['github_url'],
+                            href: '/_github/auth?redirect_uri=/funding/claim?source=' + result['github_url'],
```

Comment:
it'd be great if the username was prefilled after i got to this page http://bits.owocki.com/2M3U3m3n0k29/Screen%20Shot%202017-12-22%20at%2012.32.53%20PM.png

Reply:
(claim page)

Reply:
This was an oversight on my part.  I'll add that.